README.md

```diff
@@ -129,6 +130,44 @@ Would render without additional paragraph wrappings or unexpected title renders:
 <p> Hello, I am #1 and this is <a href='#'>my link</a>.
 ```
 
+### Markdown Plugins
+
+This reshape plugin uses [markdown-it](https://github.com/markdown-it/markdown-it) to render markdown, and so also accepts [markdown-it plugins](https://www.npmjs.com/browse/keyword/markdown-it-plugin)
+
+Many markdown-it plugins accept options as well. Typically, plugins revert to a default set of options if no options are explicitly passed.
+
+Plugins should be added to your `app.js` file, passed via `markdownPlugins:` as an array of plugins, with or without options.
```

Comment:
Since this is a reshape project, it does not necessarily need to be spike. For this section, you can just say something like "you can pass plugins through the `markdownPlugins` option as an array with or without options" 😁